wp-content/themes/workingnyc/src/js/search.js

```diff
@@ -1,65 +1,126 @@
+            allButton.className = baseButtonClass;
+            jobsButton.className = selectedButtonClass;
+            programsButton.className = baseButtonClass;
+        
+            for (var i = 0; i < searchResults.length; i++) {
+                if (searchResults[i].getAttribute("data-js-result-type") == "job") {
+                    searchResults[i].style.display = "inherit";
+                }
+                else {
+                    searchResults[i].style.display = "none";
+                }
             }
-        }
-    });
-}
+        });
+    }
 
-if (programsButton) {
-    programsButton.addEventListener("click", function () {
-        allButton.className = baseButtonClass;
-        jobsButton.className = baseButtonClass;
-        programsButton.className = selectedButtonClass;
-    
-        for (var i = 0; i < searchResults.length; i++) {
-            if (searchResults[i].getAttribute("data-js-result-type") == "program") {
-                searchResults[i].style.display = "inherit";
+    if (programsButton) {
+        programsButton.addEventListener("click", function () {
+            allButton.className = baseButtonClass;
+            jobsButton.className = baseButtonClass;
+            programsButton.className = selectedButtonClass;
+        
+            for (var i = 0; i < searchResults.length; i++) {
+                if (searchResults[i].getAttribute("data-js-result-type") == "program") {
+                    searchResults[i].style.display = "inherit";
+                }
+                else {
+                    searchResults[i].style.display = "none";
+                }
             }
-            else {
-                searchResults[i].style.display = "none";
+        });
+    }
+}
+
+else {
+    // copied from Jobs archive
+    Vue.component('SearchResult', SearchResult);
+
+    new Vue({
+    render: createElement => {
+        return createElement(SearchResultsList, {
+        props: {
+            strings: {
+            HOME: 'Home',
+            FILTERS: 'Filters',
+            CLOSE: 'Close',
+            TOGGLE_ALL: 'Toggle all {{ TERM }}',
+            CLOSE_AND_SEE_PROGRAMS: 'Close and see {{ NUMBER }} results',
+            PAGE_TITLE: 'Posts',
+            PAGE_CONTENT: '',
+            WITH: 'with',
+            SCHEDULE: 'Employment Type and Schedule',
+            SALARY: 'Salary',
+            LOCATION: 'Work Location',
+            LEARN_MORE_ABOUT: 'Learn more <span class="sr-only">about {{ PROGRAM }}</span>',
+            SHOWING: 'Showing {{ TOTAL_VISIBLE }} results of {{ TOTAL }}.',
+            RESET: 'Click here to reset filters',
+            NO_RESULTS: 'No Results. Try deselecting some filters.',
+            LOADING: 'Loading',
+            SHOW_MORE: 'Show more',
+            BACK_TO_TOP: 'Back to top',
+            SUGGEST: ''
             }
         }
-    });
+        });
+    }
+    }).$mount('[data-js="search-results-list"]');
```

Comment:
This mounts the Vue objects to the existing HTML that is rendered by WordPress in search-results.twig